prime-router/src/main/kotlin/TranslatorConfiguration.kt

```diff
@@ -57,7 +57,7 @@ abstract class TranslatorConfiguration(val type: String) : TranslatorProperties
  */
 data class Hl7Configuration
 @JsonCreator constructor(
-    // deprecated, please don't use
+    override val  schemaName: String = HL7_SCHEMA,
```

Comment:
Please add a comment over the schemaName property about why it's available and usable again, and feel free to link to the specific ticket so people can find any discussion or background.